src/include/uapi/ipc.h

```diff
@@ -817,6 +818,12 @@ struct sof_ipc_pm_ctx {
 	struct sof_ipc_pm_ctx_elem elems[];
 };
 
+/* enable or disable cores - SOF_IPC_PM_ACTIVE_CORES */
+struct sof_ipc_pm_active_cores {
```

Comment:
Best to call this sof_ipc_pm_core_config as we could eventually add other core PM data in here too.